architecture/additional_concepts/authentication.adoc

```diff
@@ -161,21 +161,20 @@ To register additional clients:
 
 ----
 $ oc create -f <(echo '
-{
-  "kind": "OAuthClient",
-  "apiVersion": "v1",
-  "metadata": {
-    "name": "demo" <1>
-  },
-  "secret": "...", <2>
-  "redirectURIs": [
-    "http://www.example.com/" <3>
-  ]
-}')
+kind: OAuthClient,
+apiVersion: v1,
+metadata:
+  name: demo <1>
+secret: "...", <2>
+redirectURIs:
+   - http://www.example.com/ <3>
+grantMethod: "...", <4>
```

Comment:
this looks like it's switching from json to invalid yaml

I would also make the sample data one of the valid values
```
grantMethod: prompt
```